EU4ToVic2Tests/HelpersTests/TechValuesTests.cpp

```diff
@@ -33,10 +33,9 @@ SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.*/
 TEST(Helpers_TechValuesTests, notValidCountryForTechConversionIfUncivilized)
 {
 	std::map<std::string, V2Country*> countries;
-	std::stringstream ideaEffectInput;
-	mappers::IdeaEffectMapper ideaEffectMapper(ideaEffectInput);
+	std::stringstream ideaEffectInput;	
```

Comment:
Can also remove this.

Reply:
Same in many following tests.

Reply:
excised ideaeffects from techvaluestests.